ydb/public/lib/ydb_cli/commands/ydb_service_scheme.cpp

```diff
@@ -543,6 +545,19 @@ int TCommandDescribe::DescribeReplication(const TDriver& driver) {
     return PrintDescription(this, OutputFormat, result, &TCommandDescribe::PrintReplicationResponsePretty);
 }
 
+int TCommandDescribe::PrintViewResponsePretty(const NYdb::NView::TDescribeViewResult& result) const {
+    Cout << "\nQuery text: " << result.GetViewDescription().GetQueryText() << Endl;
```

Comment:
The output looks like this:
- basic example, the view exists
```
...$ ydb scheme describe v
<view> v

Query text: select * from t
```
- the view does not exist
```
...$ ydb scheme describe vvv
Status: SCHEME_ERROR
Issues: 
<main>: Error: Path not found
```
- the view has [captured](https://github.com/ydb-platform/ydb/pull/8991) the `TablePathPrefix` pragma
```
...$ ydb scheme describe path/to/a/folder/v
<view> v

Query text: PRAGMA TablePathPrefix = "/Root/dedicated/path/to/a/folder";
select * from `/Root/dedicated/v`
```